clients/rust/.gitignore

```diff
@@ -1,4 +1,4 @@
 target
 # http://doc.crates.io/guide.html#cargotoml-vs-cargolock
 Cargo.lock
-src/sysadminctl.rs
+
```

Comment:
Cargo is shamelessly built around Git and Github. Neither Docs.rs nor Cargo seem to like having ignored code included in a crate, and I can understand why. 
I think the ignored file was breaking the docs and It will definitely make make the process of pushing crates automatically more annoying because it will require us to basically force push.